src/test/java/spoon/test/reference/TypeReferenceTest.java

```diff
@@ -451,11 +451,12 @@ public void testAnonymousClassesHaveAnEmptyStringForItsNameInNoClasspath() throw
 		Set<CtTypeReference<?>> referencedTypes = aClass.getReferencedTypes();
 		List<String> referencedTypeNames = referencedTypes.stream().map(Object::toString)
 				.collect(Collectors.toList());
-		assertEquals(6, referencedTypeNames.size());
+		assertEquals(7, referencedTypeNames.size());
 		assertTrue(referencedTypeNames.contains("A"));
 		assertTrue(referencedTypeNames.contains("example.B"));
 		assertTrue(referencedTypeNames.contains("java.lang.Runnable"));
 		assertTrue(referencedTypeNames.contains("java.lang.Override"));
+		assertTrue(referencedTypeNames.contains("java.lang.Object"));
```

Comment:
Seems that it is a call of Object#new Object in the implicit constructor of `new Runnable() {}`.